### PR TITLE
False positives for `match()` in `OperatorSpacingSniff`

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

**Review by jrfnl — CHANGES_REQUESTED** [review]
Nice catch! Just one remark.

File: src/Standards/Squiz/Sniffs/WhiteSpace/OperatorSpacingSniff.php [hunk 1]
```diff
@@ -77,12 +77,14 @@ public function register()
...
-            T_YIELD    => T_YIELD,
-            T_FN_ARROW => T_FN_ARROW,
+            T_RETURN        => T_RETURN,
+            T_ECHO          => T_ECHO,
+            T_EXIT          => T_EXIT,
+            T_PRINT         => T_PRINT,
+            T_YIELD         => T_YIELD,
+            T_FN_ARROW      => T_FN_ARROW,
+            T_MATCH_ARROW   => T_MATCH_ARROW,
+            T_MATCH_DEFAULT => T_MATCH_DEFAULT,
```
Comment (jrfnl) [inline src/Standards/Squiz/Sniffs/WhiteSpace/OperatorSpacingSniff.php:87]: `T_MATCH_DEFAULT` shouldn't be needed as the sniff only looks at the token directly adjacent, which will be the `T_MATCH_ARROW` between the `default` keyword and the `-`.

(oops.. apparently I'd forgotten to save my remark.... sorry)

Comment (kukulich) [reply src/Standards/Squiz/Sniffs/WhiteSpace/OperatorSpacingSniff.php]: Thanks.

**Review by jrfnl — APPROVED** [review]
👍🏻

**Comment by gsherwood** [comment]
Thanks for fixing this. I found a similar error in Squiz.Formatting.OperatorBracket as a result.

**Comment by jrfnl** [comment]
Nice!

**Pull request merged**